src/main/scala/beam/utils/scenario/urbansim/censusblock/entities/InputPlanElement.scala

```diff
@@ -16,8 +16,8 @@ case class InputPlanElement(
 
 object InputPlanElement extends EntityTransformer[InputPlanElement] {
   override def transform(m: util.Map[String, String]): InputPlanElement = {
-    val personId = getIfNotNull(m, "person_id")
-    val planElementIndex = getIfNotNull(m, "PlanElementIndex").toDouble.toInt
+    val personId = getIfNotNull(m, "person_id").split("\\.").apply(0)
+    val planElementIndex = getIfNotNull(m, "PlanElementIndex").toInt
```

Comment:
Are you sure the `toDouble` is not needed? This will throw on values such as "3.0"

Reply:
The input is the following:
![image](https://user-images.githubusercontent.com/5107562/88099327-3bc54000-cbc5-11ea-9237-6095707cec33.png)

As you can see `PlanElementIndex` is an integer